src/CheckBoxFilter.webmodeler.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, createElement } from "react";
+import * as classNames from "classnames";
+
+import { Alert } from "./components/Alert";
+import { CheckboxFilter } from "./components/CheckBoxFilter";
+import { ContainerProps } from "./components/CheckBoxFilterContainer";
+import { Utils, parseStyle } from "./utils/ContainerUtils";
+
+// tslint:disable-next-line class-name
+export class preview extends Component<ContainerProps, {}> {
+    constructor(props: ContainerProps) {
+        super(props);
+    }
+
+    render() {
+        return createElement("div",
+            {
+                className: classNames("widget-checkbox-filter", this.props.class),
+                style: parseStyle(this.props.style)
+            },
+            this.renderAlert(),
+            createElement(CheckboxFilter, {
+                handleChange:  () => { return; },
+                isChecked: this.props.defaultChecked
+            })
+        );
+    }
+
+    private renderAlert() {
```

Comment:
Most likely this will throw an error since you are doing a validation yet the list view is not yet there. Just cross check
I suggest you add the Alert as part of the render and use state to update the message

Reply:
In the web-modeler we always have to display both the validation-message and the widget's component. So this is the appropriate.